### ISSUE TITLE
Running OpenJazz in 1920x1080, HUD Missing

### ISSUE THREAD

**Issue by Gagert** [AlisterT/openjazz#76]
I just tried the released build of this project on my JazzCD install, and upon setting the game to run in fullscreen, setting the resolution to 1920x1080 (my native screen resolution) and get a stage up & running, the HUD disappeared. The HUD reappears once I lower the Screen Resolution to something like 1680x960. I prefer to use 3X scale, as I feel that is the perfect middleground to achieve a good field of view, without everything looking too small. I'm using Windows 10 64 bit.

Edit: Playing a bit more with the settings, it really seems like anything beyond a resolution of 1680x960 makes the scaling go bananas, thereby objects appearing beyond me screen area. Thereby using the same screen resolution as my monitor, 1920x1080, makes things at the very end of the screen, both up, down, left & right disappear.

Here's the game running at 1680x960:

![](https://i.imgur.com/RpBXQXn.png)

Here's the game running at 1920x1080:

![](https://i.imgur.com/gSYLdjv.png)

When running the game at 1920x1080, I can't actually see Jazz at the very start of the level, as he lands on the patch of grass to the left that's not visible in the picture at OpenJazz running at 1680x960.

This might have something to do with the Scale2x setting OpenJazz probably defaults to.

**Comment by carstene1ns** [comment]
I cannot really reproduce this. Works for me.

**Comment by Gagert** [comment]
I found the issue, it's Windows 10 related. If you use a higher DPI setting on your desktop (I'm using my TV now, so I prefer higher DPI zoom), applications may get affected as well. The solution was to right click and enter properties on OpenJazz.exe, and do the following:

![](https://i.imgur.com/Jm6HCci.png)

I've had to do the same procedure with other apps as well, I just forgot about this possibly being the reason for this issue yesterday. Closing this issue.
